public/swagger

```diff
@@ -0,0 +1 @@
+Subproject commit 86be831a8b4ef73ce0a83c94fec93a526994ef95
```

Comment:
The swagger ui is a separate project which we are referencing, not including the source code of - I didn't add that line explicitly (though I did add a sub project reference) but I'm guessing this means we reference a specific commit of the swagger ui project